CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+
+If the pull request is still work in progress, please open the pull request in draft mode.
+
+### Testing
+
+Testing is a key component in the development workflow. We run automated tests on all pull requests to enforce strict linting rules and also make sure that changes introduced to the IaC are not introducing code degredation.
+
+### Code Review
+
+If you want the team to review the changes, mark the pull request as "Ready for review". The team will get back to you and approve the PR, provide comments or request changes if necessary. The review process may take a few cycles in order to ensure that all content is of high quality and well structured.
+
+### Merge
+
+After the pull request has been reviewed and approved, the feature branch will be merged into the main branch and automatically closed and deleted.
+
+## Steering Committee and Stewardship of the project
+
+The Data Management & Analytics Scenario Committee and its members (aka Steering Committee Members) are the primary caretakers of the Data Management & Analytics Scenario repos including language, design, policy oversight, and reference implementations. Membership to steering committee will be solely baesd on active contribution and thought leadership in the preject and community.
+
+Voting members may be added once approved by the Steering Committee. All members abide by all organizational polices, including the code of conduct. The membership of a Steering Committee member can be withdrawn in the case of written notice of resignation or if the member is unreachable or unresponsive for more than three months.
+
+The following users are Steering Committee Members:
+
+| **NAME**             | **GitHub Handle** | **Affiliated Organization** |
+|:---------------------|:------------------|:----------------------------|
+| Marvin Buss          | @marvinbuss       | Microsoft                   |
+| Espen Brandt-Kjelsen | @esbran           | Microsoft                   |
+| Uday Pandya          | @uday31in         | Microsoft                   |
+| Amanjeet Singh       | @amanjeetsingh    | Microsoft                   |
+| Andre Hass           | @andrehass        | Microsoft                   |
+| Roberto Cavalcanti   | @rocavalc         | Microsoft                   |
+| Sanem Sever          | @sasever          | Microsoft                   |
+
+### Steering Committee Member Responsibilities
+
+Committee Members are responsible for reviewing and approving RFCs proposing new features or design changes. The initial Committee consists of Microsoft employees. It is expected that over time, community will grow and new community members will join Committee Members. Membership is heavily dependent on the level of contribution and expertise: individuals who contribute in meaningful ways to the project will be recognized accordingly.
```

Comment:
Consider changing this: 
_individuals who contribute in meaningful ways to the project will be recognized accordingly._

To this:
individuals who actively contribute in a meaningful way over time and has the necessary skills to take on additional responsibility will be considered.